scalingo/utils_slice.go

```diff
@@ -1,5 +1,18 @@
 package scalingo
 
+// Filter filters slice by using the test function.
+// It returns a filtered slice.
+func Filter[S any](slice []S, test func(S) bool) []S {
```

Comment:
suggestion: I like the Ruby wording for this kind of function (https://devdocs.io/ruby~2.7/array#method-i-keep_if). What do you think about renaming it `keepIf` or `select`?

Reply:
I'd like `select` but it is a key word in Go already.
I'm not really fan of `keepIf`, I don't find it really consistent with the Go syntax. Maybe if it had been a method rather than a function.
But anyway, let's go with that, because `filter` is clearly worse.